README.md

```diff
@@ -27,27 +27,88 @@ cd ./devop-tools/docker/data-source-services && docker-compose up --build -d
 ```
 
 #### Specific Setup - Windows
+
 * Kill, and optionally disable, the `World Wide Web Publishing Service`
 * Run `set COMPOSE_CONVERT_WINDOWS_PATHS=1` in your CMD or PowerShell terminal
 * Optionally, [read](https://github.com/docker/compose/issues/4303#issuecomment-379563170) this bug report.
 
 ### Optional: Data Source Tools
+
 * Optionally included is the following tools:
-  * phpMyAdmin
+* phpMyAdmin
 * `cd ./devop-tools/docker/data-source-tools && docker-compose up --build -d`
 
 ## phpMyAdmin
+
 If the optional tools are launched, you can find phpMyAdmin at: localhost:8080
+
 * It supports the following databases...
-  * mariadb106 (lts)
-  * mariadb1011 (lts)
+* mariadb106 (lts)
+* mariadb1011 (lts)
 
 ## Scripts
+
 Inside the `scripts` folder you will find one-off scripts to help with tasks.
 
 * `db_migrate.sh` - Helps migrate databases between versions of mysql.
 
+## CLI Tooling Setup
+
+Add the `bin` directory to your path to use **Hop** and **Git Scripts**.
+
+## Hop
+
+MISSING DESCRIPTION
+
+## Git Scripts
+
+Custom scripts that extend Git functionality, to streamline the process of tracking tickets and managing releases
+
+### `git-tickets`
+
+This command is used to get the tickets since staging was last updated.
+By default, it does not update the branches.
+
+```shell
+git tickets [options] [arguments]
```

Comment:
```suggestion
git-tickets [options] [arguments]
```

Reply:
This is not how you'd run them having scripts prefixed with `git-` makes it a git sub command so `git tickets` and `git make-release` is valid